src/borg/_hashindex.c

```diff
@@ -27,6 +27,17 @@
 #define MAGIC "BORG_IDX"
 #define MAGIC_LEN 8
 
+#define DEBUG 0
+
+#define debug_print(fmt, ...)                   \
+  do {                                          \
+    if (DEBUG) {                                \
+      fprintf(stderr, fmt, __VA_ARGS__);        \
+      fflush(NULL);                             \
+    }                                           \
+  } while (0)
```

Comment:
hmm, the do..while is so you just get a block?

wouldn't using {..} also work for that purpose?